bin/oref0-setup.sh

```diff
@@ -194,6 +214,10 @@ if openaps vendor add --path . mmeowlink.vendors.mmeowlink 2>&1 | grep "No modul
         echo -n "Cloning mmeowlink dev: "
         (cd ~/src && git clone -b dev git://github.com/oskarpearson/mmeowlink.git) || die "Couldn't clone mmeowlink dev"
     fi
+	if [[ "$radio_locale" -eq "WW" ]]; then
+		echo -n "Patching mmtune.py for WW frequencies. " # temporary fix for https://github.com/oskarpearson/mmeowlink/issues/16
+		sed -i "s/def __init__(self, link, pumpserial, radio_locale='US'):/def __init__(self, link, pumpserial, radio_locale='WW'):/g" ~/src/mmeowlink/mmeowlink/mmtune.py || die "Couldn't patch mmtune.py"
```

Comment:
I'd really like to avoid doing this via patch if possible. The mmeowlink code can be changed to make the parameter available as a config field instead... it's just not something I've got to yet.

Could we pause this for the moment and I'll try have a look one of the evenings this week? My time is a bit limited at the moment as I'm moving house.